harness/bin/test_harness_driver.py

```diff
@@ -338,7 +338,7 @@ def test_harness_driver(argv=None):
 
     resubmit_count = -1 # -1 means resubmit forever until stopped
     if do_submit:
```

Comment:
Switching to equality operators seems like the correct change, but should this be != instead? I find separating the logical operators to be potentially confusing.